airflow-core/src/airflow/api_fastapi/auth/managers/simple/ui/src/login/LoginForm.tsx

```diff
@@ -53,7 +53,8 @@ export const LoginForm = ({ isPending, onLogin }: LoginFormProps) => {
           render={({ field, fieldState }) => (
             <Field.Root invalid={Boolean(fieldState.error)} required>
               <Field.Label>Username</Field.Label>
-              <Input {...field} />
+              {/* eslint-disable-next-line jsx-a11y/no-autofocus */}
+              <Input autoFocus={true} {...field} />
```

Comment:
If my front-end skills are not entirely lost I dont think we need `={true}`? I might be wrong

```suggestion
              <Input autoFocus {...field} />
```

Reply:
+1

Reply:
Ah I was not realizing maybe earlier that the error was also regarding the linting. Yeah thanks, optimized the chars of truth away :-D